wicket/tests/output/example_non_empty.toml

```diff
@@ -50,6 +50,17 @@ bootstrap_sleds = [
     5, # serial 4 5 6 (model model2 revision 5, ::1)
 ]
 
+# Allowlist of source IPs that can make requests to user-facing services.
+[allowed_source_ips]
+# Any external IPs to make requests. This is the default.
+allow = "any"
```

Comment:
This is an algebraic bounded lattice similar to the BGP prefix allowlist below (a list of some values + all). Would it be possible to align the way this is defined with the BGP prefixes below?

https://en.m.wikipedia.org/wiki/Lattice_(order)

The wicket bits aren't blocking for r8 since it's user facing config and not part of the stable API. But it would be nice for the bootstore configs to be aligned. (But again not a huge deal, we can migrate the JSON configs fine.)

Reply:
Yeah, I'd prefer to do that in a follow-up, since this has already been a long road. I'd like some time to read through the use of the BGP allow-list type, too. It looks like the semantics might be slightly different, so I'd like to look it over in more detail.